client/agora/views/dashboard/partials/gallery/note-gallery.ejs

```diff
@@ -82,43 +83,100 @@
           </div>
 
         <% }}%>
-          <h1 class="text-secondary pt-3 pl-1">My Notes</h1>
-          <hr>
+        
+
+        
+        <div id="gallery-header">
+          <h1 class="text-secondary pt-3 pl-1 pb-0">My Notes</h1>
+          <div class="btn-group btn-group-toggle btn-goal-topic-toggle mr-5" data-toggle="buttons" >
+        <label class="btn btn-secondary active">
+         <input  type="radio" name="options" id="option1"  autocomplete="off" checked onclick="toggleGoalView()"> Goal
+          </label>
+        <label class="btn btn-secondary">
+          <input type="radio" name="options" id="option2" autocomplete="off" onclick="toggleTopicView()" > Topic
+        </label>
+         <label class="btn btn-secondary">
+         <input type="radio" name="options" id="option3" autocomplete="off" onclick="toggleAllView()"> All
+        </label>
+</div>
+</div>
+          <hr class="mt-0">
           <div class="container">
 
 
             <% for (let i = 0; i < availableResources.length + 1; i++) { 
               //add button
               if (i === 0) {
                  %>
-                <div class="row ">
+                <div class="row row-cols-1 row-cols-md-3 g-4 ">
                   <div class="col mb-3 gallery-col align-items-stretch">
                     <div class="card h-100" data-toggle="modal" data-target="#new-topic-goal-modal"      >
-                      <div class="card-body d-flex flex-column" style="margin-top: 8%">
+                      <div class="card-body d-flex flex-column" style="margin-top: 8%" id="add">
                         <img src="/assets/img/buttons/new-content.png" class="mx-auto d-block btn-add-content"
                           alt="Add content">
                       </div>
                     </div>
                   </div>
-
-                <% }else if(i % 3===0) {%>
-              </div>
-              <div class="row ">
-                <% renderCard(i-1,availableResources[i-1].resourceName,availableResources[i-1].resourceDescription,["Bio","Chem"],"Goal", ["Doc 1","Doc 2","Doc 3", "Doc 4","Doc 5"])
-                  %>
-
-
-                  <% }else{ %>
+                  <% }else{ 
+                    if(i % 2 === 0) {%>
                     <% renderCard(i,availableResources[i-1].resourceName,availableResources[i-1].resourceDescription,["Bio","Chem"],"Note", null)
-                      %>
+                    }else{
+                      renderCard(i-1,availableResources[i-1].resourceName,availableResources[i-1].resourceDescription,["Bio","Chem"],"Goal", ["Doc 1","Doc 2","Doc 3", "Doc 4","Doc 5"])
+                     } %>
                       <% }}%>
               </div>
+            
 
 
 
              
 
                 <script>
```

Comment:
No inline scripts!!! :)
all JavaScript MUST be in .js file!

Reply:
Ok sounds good just resolved that. I am creating a task for changing jquery to js so the person who wrote code can covert what the have done. However the subject of the pull request has been adjusted to be in the dashboard.js file.